src/Repository/ProductCommentRepository.php

```diff
@@ -166,6 +166,43 @@ public function getAverageGrade($productId, $validatedOnly)
         return (float) $qb->execute()->fetchColumn();
     }
 
+    /**
+     * @param array $productIds
+     * @param bool $validatedOnly
+     *
+     * @return array
+     */
+    public function getAverageGrades(array $productIds, $validatedOnly)
+    {
+        $sql = 'SELECT';
+
+        $count = count($productIds);
+
+        foreach ($productIds as $index => $id) {
+            $sql .= " SUM(IF(id_product = '" . $id . "' AND deleted = 0";
```

Comment:
```suggestion
            $sql .= ' SUM(IF(id_product = ' . pSQL($id) . ' AND deleted = 0';
```
This is what I wanted to say :sweat_smile: 
You need to escape value to prevent sql injections

Reply:
Espacing value added, double quote replaced with single. I am sorry that I wasted your time because of my lack of knowledge. I don't know that well Prestashop yet.

Reply:
You're not wasting my time, you did a great job with this PR. You saved us times :)
We really appreciate this kind of contribution :heart:

Reply:
Thanks a lot @Oksydan & @PierreRambaud